src/py42/exceptions.py

```diff
@@ -167,6 +167,13 @@ def __init__(self, exception, message=None):
         super(Py42UserAlreadyExistsError, self).__init__(exception, message)
 
 
+class Py42UserDoesNotExist(Py42NotFoundError):
```

Comment:
I think the name of the exception is misleading. It makes it sound like either the user does not exist in Code42 or the user does not have a detection list profile.

A user is able to have a detection list profile and still not be on a detection list.

That makes it likely to get confused with Code42 CLI exception `UserDoesNotExistError`.

We should rename this to be something like `Py42UserNotOnListError`.

Reply:
Changed.